packages/cardpay-sdk/sdk/scheduled-payment-module.ts

```diff
@@ -735,8 +735,8 @@ export default class ScheduledPaymentModule {
     recurringUntil?: number | null,
     onScheduledPaymentCreate?: (scheduledPaymentId: string) => unknown
   ) {
-    let hubAuth = await getSDK('HubAuth', this.web3);
-    let hubRootUrl = await hubAuth.getHubUrl();
+    let hubAuth = await getSDK('HubAuth', this.web3, 'http://localhost:3000');
```

Comment:
how will this API work when called from the CLI with this hard coded hub endpoint?

Reply:
Good catch - forgot to remove this argument.